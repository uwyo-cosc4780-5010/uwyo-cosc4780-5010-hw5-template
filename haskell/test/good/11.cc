// Macros to shut up the C++ warnings
#define printInt(x)
#define printDouble(x)
#define printString(x)
#define readInt() 0
#define readDouble() 0
#define readString() 0

#include <string>

using std::string;

bool callMeMaybe(bool x)
{
    return x;
}

int main()
{
    int x = 10;
    double y = 50.0;
    callMeMaybe((10 > 5) || ((y = 3.0) > 1.0));
    if (y == 50.0)
    {
        printString("nice distance\n");
    }
    else
    {
        printString("whoopsie!\n");
    }
    callMeMaybe((10 < 5) && ((y = 3.0) > 1.0));
    if (y == 50.0)
    {
        printString("epic hacking\n");
        printString("B)\n");
    }
    else
    {
        printString("oh no!\n");
        printString(":,(\n");
    }
    return 0;
}