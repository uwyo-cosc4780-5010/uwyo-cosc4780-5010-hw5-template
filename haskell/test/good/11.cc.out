nice distance
epic hacking
B)
